src/main/java/io/javalin/core/plugin/PluginLifecyles.kt

```diff
@@ -1,13 +1,22 @@
 package io.javalin.core.plugin
 
 import io.javalin.Javalin
+import org.eclipse.jetty.server.Server
```

Comment:
I'm wondering if this should instead be `JavalinServer` ?

Reply:
Yeah, `JavalinServer` is probably a better solution

Reply:
I had the problem that `JavalinServer` was protected on the app. To access it I created a getter. With this getter it is not necessary anymore to have a separate lifecycle for this, so I removed it. What do you think about this change?